crates/client-api/src/routes/database.rs

```diff
@@ -409,27 +403,17 @@ pub struct LogsQuery {
     follow: bool,
 }
 
-fn auth_or_unauth(auth: SpacetimeAuthHeader) -> axum::response::Result<SpacetimeAuth> {
-    auth.get()
-        .ok_or((StatusCode::UNAUTHORIZED, "Invalid credentials").into())
-}
```

Comment:
I can't tell if I'm reading this correctly. Did this use to effectively require a pre-existing auth? If so it feels like the usages below should switch to `Required`?